Libs/MRML/Core/vtkMRMLSliceNode.h

```diff
@@ -78,11 +78,27 @@ class VTK_MRML_EXPORT vtkMRMLSliceNode : public vtkMRMLAbstractViewNode
   vtkGetMacro(SliceVisible, int);
   vtkSetMacro(SliceVisible, int);
 
+  ///
+  /// The visibility of the slice edge in the 3DViewer.
+  vtkGetMacro(ThreeDSliceEdgeVisibility, int);
```

Comment:
Use `SliceEdgeVisibility3D` instead, it is more similar to `SliceVisible` and this suffix is used elsewhere similarly, too.